openlibrary/templates/lists/home.html

```diff
@@ -36,7 +36,7 @@ <h2>$_('Active Lists')</h2>
                         <div class="cover">
                             $ cover = list.get_cover() or list.get_default_cover()
                             $ cover_url = cover and cover.url("S") or "/images/icons/avatar_book-sm.png"
-                            <img src="$cover_url" alt=""/>
+                            <img src="$cover_url" alt="$_('List cover')"/>
```

Comment:
Purely presentational, so no need for screenreader alt text here!
```suggestion
                            <img src="$cover_url" alt=""/>
```